website/docs/components/card/partials/guidelines/guidelines.md

```diff
@@ -8,29 +8,45 @@ Use a card container to help build more consistently styled cards.
 
 ### Static cards
 
-Static cards may have interactive elements within them but are not actionable or interactive on their own.
+Static cards may have interactive elements within them but are not actionable or interactive on their own. Card should not have any elevations applied to it to appear visually flat.
+
+![A static card titled "Billing summary" with an info notification and a link.](/assets/components/card/card-static-flat-looking.png)
 
 ### Interactive cards
 
-Interactive cards are actionable (e.g., it can be hovered, clicked, focused, etc.)
+Interactive cards are actionable, e.g., they can be hovered over, clicked, or focused on. If a card is interactive, it should not include additional interactive elements within it.
```

Comment:
```suggestion
Interactive cards are actionable, e.g., they can be hovered over, clicked, and focused on. If a card is interactive, it should not include additional interactive elements within it.
```

In order for the interaction to be accessible, it needs to be all 3. Like they can't just apply hover styles bc that implies that it is only usable with a mouse.

Reply:
@majedelass It looks like this comment got missed.